docs/rules/prefer-enabled-disabled.md

```diff
@@ -0,0 +1,30 @@
+# prefer toBeDisabled() or toBeEnabled() over toHaveProperty('disabled', true|false) (prefer-enabled-disabled)
+
+## Rule Details
+
+This rule aims to improve readability of tests.
```

Comment:
Can you elaborate little bit more? Also, specify this rule is supposed to be use together with `@testing-library/jest-dom` package.

Reply:
Will do

Reply:
done.